packages/style-utilities/src/styles/getFocusStyle.ts

```diff
@@ -94,6 +99,7 @@ function _getFocusStyleInternal(theme: ITheme, options: IGetFocusStylesOptions =
         border: `${width}px solid ${borderColor}`,
         outline: `${width}px solid ${outlineColor}`,
         zIndex: ZIndexes.FocusStyle,
+        borderRadius: borderRadius,
```

Comment:
Does this also allow [the shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius#syntax), e.g.:

`border-radius: 1px 2px 3px 4px`?

Reply:
it allows string or number

Reply:
![image](https://user-images.githubusercontent.com/1434956/187741463-7f23000c-33c9-467b-86bf-89fe7eb7bde4.png)